### ISSUE TITLE
Exception thrown during stop

### ISSUE THREAD

**Issue by rvega-arg** [allegro/embedded-elasticsearch#26]
Using elasticsearch:5.3.3 and embedded-elasticsearch:2.4.1 the following exception is thrown when stop() is called.

```
	private EmbeddedElastic elastic;

	@Before
	public void before() throws IOException, InterruptedException {
		elastic = EmbeddedElastic.builder().withElasticVersion("5.5.3").withStartTimeout(1, MINUTES).build().start();
	}

	@After
	public void after() {
		elastic.stop();
	}
```

```
Exception in thread "EmbeddedElsHandler" pl.allegro.tech.embeddedelasticsearch.EmbeddedElasticsearchStartupException: java.io.IOException: Stream closed
	at pl.allegro.tech.embeddedelasticsearch.ElasticServer.lambda$startElasticProcess$7(ElasticServer.java:89)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: Stream closed
	at java.io.BufferedInputStream.getBufIfOpen(BufferedInputStream.java:170)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:283)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.io.InputStreamReader.read(InputStreamReader.java:184)
	at java.io.BufferedReader.fill(BufferedReader.java:161)
09:21:21.258 [ElsInstanceCleaner] INFO  p.a.t.e.ElasticServer - Stopping elasticsearch server...
	at java.io.BufferedReader.readLine(BufferedReader.java:324)
	at java.io.BufferedReader.readLine(BufferedReader.java:389)
	at pl.allegro.tech.embeddedelasticsearch.ElasticServer.lambda$startElasticProcess$7(ElasticServer.java:84)
	... 1 more
```

**Comment by todvora** [comment]
Hi, 
encountered the same issue, using elastic 5.6.0. 

The very same exception is recorded also in logs from Travis-CI builds: https://travis-ci.org/allegro/embedded-elasticsearch/builds/277167816#L831

I have tried to solve it in https://github.com/allegro/embedded-elasticsearch/pull/30, which seems to work fine.

**Comment by djarosz** [comment]
I think this might be introduced by this PR [#24](https://github.com/allegro/embedded-elasticsearch/pull/24)

**Comment by gaczm** [comment]
I've released fix from #30 as version 2.4.2. It should be available in Maven Central soon

**Comment by todvora** [comment]
@gaczm Perfect, thank you!
